lego/apps/users/fixtures/test_abakus_groups.yaml

```diff
@@ -173,3 +173,8 @@
   fields: {deleted: false, name: Students, description: '', contact_email: '', parent: null,
     logo: null, type: annen, text: '', permissions: '[]', show_badge: true, lft: 1,
     rght: 2, tree_id: 17, level: 0}
+- model: users.abakusgroup
```

Comment:
This file is generated by this file: https://github.com/webkom/lego/blob/474ae2536507e304a87e007e542e0710ba24b57e/lego/apps/users/fixtures/test_abakus_groups.py (as you see on the first few lines :smile:)